src/components/implementation/tests/micro_booter/micro_booter.c

```diff
@@ -39,7 +41,17 @@ cos_init(void)
 	termthd[cos_cpuid()] = cos_thd_alloc(&booter_info, booter_info.comp_cap, term_fn, NULL);
 	assert(termthd[cos_cpuid()]);
 	PRINTC("Micro Booter started.\n");
-	test_run_mb();
+
+    // BOOTER TESTS
+    //test_run_mb();
+
+    // IPI TESTS
```

Comment:
Yeah, I'd move them to a different folder. micro_xcore or something.

Reply:
Fixed